_posts/2020-07-23-compchem-at-bcc.md

```diff
@@ -0,0 +1,12 @@
+---
+site: freiburg
+title: "Computational chemistry at BCC2020"
+supporters:
+ - galaxy-europe
+---
+
+Several sessions were related to computational chemistry during this year's BCC. On July 18th we ran a successful introductory tutorial on 'Intuitive, reproducible high-throughput molecular dynamics in Galaxy'. The online training was led jointly by representatives from the European Galaxy Team (Simon Bray) and the University of Cape Town (Chris Barnett, Tharindu Senapathi).
+
+We also heard more during the conference from Chris about the work being carried out at the University of Cape Town on antibody-antigen binding patterns.
+
```

Comment:
Does it work when you include `<iframe title="vimeo-player" src="https://player.vimeo.com/video/437703080" width="640" height="360" frameborder="0" allowfullscreen></iframe>`